src/main/kotlin/no/nav/familie/ef/sak/behandling/fremleggsoppgave/FremleggsoppgaveReporitory.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.ef.sak.behandling.fremleggsoppgave
+
+import no.nav.familie.ef.sak.repository.InsertUpdateRepository
+import no.nav.familie.ef.sak.repository.RepositoryInterface
+import org.springframework.stereotype.Repository
+import java.util.UUID
+
+@Repository
+interface FremleggsoppgaveReporitory :
```

Comment:
```suggestion
interface FremleggsoppgaveRepository :
```
Navnet på filen må også endres